relational/table.py

```diff
@@ -229,6 +231,35 @@ def __init__(self):
             self.NET_EST_NUM_UNITS_BMR_DATA,
         ])
 
+    def _maybe_add_geo(self, row):
+        """Adds 'San Francisco, CA' to an address if necessary.
+
+        PowerBI seems to handle address queries not very well, so even a
+        fairly unambiguous '1 South Park 94107' ends up being placed in the
+        UK.  The best way to deal with this is explicitly specify
+        San Francisco, CA."""
+        addr = row[self.index(self.ADDRESS)]
+        if not addr:
+            return
+
+        if re.search('san francisco', addr, flags=re.IGNORECASE):
```

Comment:
haven't seen it in the data.

honestly am still not sure about this PR. it helps the map issue a bit, but i'm still getting wild locations. so not sure if this is value-adding since it doesn't actually seem to fix bing's apparently not-great location finding.

Reply:
Hmmmm, admittedly I'm a little hesitant using regexes like this for addresses. Is this something the address normalization library steven was using can help with? It feels like if there's a simple thing here that gets us a big win (i.e. just adding san francisco, ca everywhere that needs it) maybe its worthwhile but going down a rathole of fixing addresses seems like it might not be worth the time we put in?